explanation/cryptography/crypto_dashboard.md

```diff
@@ -48,6 +48,6 @@ The dashboard uses mutual TLS authentication to establish a trusted TLS communic
 
 ## Packages used
 
-* [PyJWT](https://github.com/jpadilla/pyjwt)
-* [`python-jose`](https://github.com/mpdavis/python-jose/)
-* [OpenSSL](https://launchpad.net/ubuntu/+source/openssl/)
\ No newline at end of file
+* [PyJWT](https://github.com/jpadilla/pyjwt), supplied by [PyPI](https://pypi.org/project/PyJWT/)
+* [`python-jose`](https://github.com/mpdavis/python-jose/), supplied by [PyPI](https://pypi.org/project/python-jose/)
+* [OpenSSL](https://launchpad.net/ubuntu/+source/openssl/), preloaded in the [ubuntu:jammy](https://hub.docker.com/layers/library/ubuntu/jammy/images/sha256-94db6b944510db19c0ff5eb13281cf166abfe6f9e01a6f8e716e976664537c60?context=explore) base image
```

Comment:
```suggestion
* [OpenSSL](https://launchpad.net/ubuntu/+source/openssl/), preloaded in the [Ubuntu 22.04 (Jammy Jellyfish)](https://hub.docker.com/layers/library/ubuntu/jammy/images/sha256-94db6b944510db19c0ff5eb13281cf166abfe6f9e01a6f8e716e976664537c60?context=explore) base image
```

Reply:
Why do we link docker hub here? We're not using any docker image as part of the artifacts we deliver.

If we just put `* [OpenSSL](https://launchpad.net/ubuntu/+source/openssl/)` as we do for the other explanations, I think that is sufficient enough. Calling out a specific verison/Ubuntu series is subject to the SBOM.

Reply:
Addressed the [changes](https://github.com/canonical/anbox-cloud-docs/compare/0a40d7f5ae597e540142633bac1eab60df107602..6aa9e194455d68eb5df6fa28c4fe753e39092ab3)